superduperdb/vector_search/atlas.py

```diff
@@ -31,8 +31,10 @@ def __init__(
         **kwargs,
     ):
         self.identifier = identifier
-        db_name = CFG.cluster.vector_search.split('/')[-1]
-        self.database = getattr(pymongo.MongoClient(CFG.cluster.vector_search), db_name)
+        vector_search_uri = CFG.cluster.vector_search.uri
+        assert vector_search_uri, 'Vector search URI is required'
+        db_name = vector_search_uri.split('/')[-1]
+        self.database = getattr(pymongo.MongoClient(vector_search_uri), db_name)
```

Comment:
After I added the type reminder for Config, an error occurred during detection here. It may be because we changed the structure of config and did not update it here. I fixed it, but did not test it.